pages/design/atomic-template-empty.md

```diff
@@ -0,0 +1,82 @@
+---
+name:
+test_type: atomic
+
+description: |
+  This rule checks ...
+
+success_criterion:
+- x.x.x
+
+test_aspects:
+- 
+
+authors:
+-
```

Comment:
Add "(full names as found in https://github.com/auto-wcag/auto-wcag/blob/master/_data/contributors.yml - if not yet listed, please have authors added to the list)" ?